pkg/kubelet/kubelet.go

```diff
@@ -551,6 +551,7 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	if err != nil {
 		return nil, err
 	}
+	machineInfo.Timestamp = time.Time{}
```

Comment:
Just to refresh my memory, does this cause timestamps not to be set for metrics, or does it set the timestamp at 0?

Reply:
The collector will not include a timestamp because it is set to zero: https://github.com/google/cadvisor/blob/65fa5b44d3840d465a5c004a0f6722e0fd540137/metrics/prometheus_machine.go#L240

Reply:
makes sense.  A comment would be helpful for future readers.

Reply:
Added. Leaving this unresolved so others can find this.